core/common/src/main/java/alluxio/util/io/FileUtils.java

```diff
@@ -71,16 +74,20 @@ public static void changeLocalFileGroup(String path, String group) throws IOExce
    * @param filePath that will change permission
    * @param perms the permission, e.g. "rwxr--r--"
    */
-  public static void changeLocalFilePermission(String filePath, String perms) throws IOException {
-    Files.setPosixFilePermissions(Paths.get(filePath), PosixFilePermissions.fromString(perms));
+  public static void changeLocalFilePermission(String filePath, String perms) {
+    try {
+      Files.setPosixFilePermissions(Paths.get(filePath), PosixFilePermissions.fromString(perms));
+    } catch (IOException e) {
```

Comment:
You can catch specific exceptions as defined in the function API and handle them here. the implementation of `AlluxioRuntimeException.from` should be simple.